README.md

```diff
@@ -1 +1,24 @@
 # hw01
+
+# Adeline Kao Personal Biography
+## Introduction
+**Hello! My name is Adeline Kao and this is my short personal biography.**
+*(I go by the pronouns she, her, hers.)*
+## Personal Biography
+![Image of Adeline Kao](Adeline_Kao_Picture.png)
```

Comment:
Since the file "Adeline_Kao_Picture.png" refers to a local path, other people would not be able to see the photo if they only download the readme.md file. I see that the rendering is fine because now the md file and jpg file are in the same folder in the repo. But the path, https://github.com/AdelineKao/hw01/blob/master/Adeline_Kao_Picture.png?raw=true is the URL address for the uploaded photo file.